v2/api/apimanagement/v1api20220801/structure.txt

```diff
@@ -0,0 +1,571 @@
+│   │   │   │       ├── "Consumption"
+│   │   │   │       ├── "Developer"
+│   │   │   │       ├── "Isolated"
+│   │   │   │       ├── "Premium"
+│   │   │   │       └── "Standard"
+│   │   │   ├── VirtualNetworkConfiguration: *Object (1 property)
+│   │   │   │   └── SubnetResourceReference: *genruntime.ResourceReference
+│   │   │   └── Zones: string[]
+│   │   ├── ApiVersionConstraint: *Object (1 property)
+│   │   │   └── MinApiVersion: *string
+│   │   ├── AzureName: Validated<string> (3 rules)
+│   │   │   ├── Rule 0: MaxLength: 50
+│   │   │   ├── Rule 1: MinLength: 1
+│   │   │   └── Rule 2: Pattern: "^[a-zA-Z](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?$"
+│   │   ├── Certificates: Object (4 properties)[]
+│   │   │   ├── Certificate: *Object (3 properties)
+│   │   │   │   ├── Expiry: *string
+│   │   │   │   ├── Subject: *string
+│   │   │   │   └── Thumbprint: *string
+│   │   │   ├── CertificatePassword: *string
+│   │   │   ├── EncodedCertificate: *string
+│   │   │   └── StoreName: *Enum (2 values)
+│   │   │       ├── "CertificateAuthority"
+│   │   │       └── "Root"
+│   │   ├── CustomProperties: map[string]string
+│   │   ├── DisableGateway: *bool
+│   │   ├── EnableClientCertificate: *bool
+│   │   ├── HostnameConfigurations: Object (11 properties)[]
+│   │   │   ├── Certificate: *Object (3 properties)
+│   │   │   │   ├── Expiry: *string
+│   │   │   │   ├── Subject: *string
+│   │   │   │   └── Thumbprint: *string
+│   │   │   ├── CertificatePassword: *string
+│   │   │   ├── CertificateSource: *Enum (4 values)
+│   │   │   │   ├── "BuiltIn"
+│   │   │   │   ├── "Custom"
+│   │   │   │   ├── "KeyVault"
+│   │   │   │   └── "Managed"
+│   │   │   ├── CertificateStatus: *Enum (3 values)
+│   │   │   │   ├── "Completed"
+│   │   │   │   ├── "Failed"
+│   │   │   │   └── "InProgress"
+│   │   │   ├── DefaultSslBinding: *bool
+│   │   │   ├── EncodedCertificate: *string
+│   │   │   ├── HostName: *string
+│   │   │   ├── IdentityClientId: *string
+│   │   │   ├── KeyVaultId: *string
+│   │   │   ├── NegotiateClientCertificate: *bool
+│   │   │   └── Type: *Enum (5 values)
+│   │   │       ├── "DeveloperPortal"
+│   │   │       ├── "Management"
+│   │   │       ├── "Portal"
+│   │   │       ├── "Proxy"
+│   │   │       └── "Scm"
+│   │   ├── Identity: *Object (2 properties)
+│   │   │   ├── Type: *Enum (4 values)
+│   │   │   │   ├── "None"
+│   │   │   │   ├── "SystemAssigned"
+│   │   │   │   ├── "SystemAssigned, UserAssigned"
+│   │   │   │   └── "UserAssigned"
+│   │   │   └── UserAssignedIdentities: Object (1 property)[]
+│   │   │       └── Reference: genruntime.ResourceReference
+│   │   ├── Location: *string
+│   │   ├── NatGatewayState: *Enum (2 values)
+│   │   │   ├── "Disabled"
+│   │   │   └── "Enabled"
+│   │   ├── NotificationSenderEmail: Validated<*string> (1 rule)
+│   │   │   └── Rule 0: MaxLength: 100
+│   │   ├── Owner: *genruntime.KnownResourceReference
+│   │   ├── PrivateEndpointConnections: Object (4 properties)[]
```

Comment:
I think that this whole property is actually readonly.

Reply:
Done